projects/PM_hash_navigation/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.1)
+include(../../cmake/bob_robotics.cmake)
+BoB_project(SOURCES main.cc
+            EXTERNAL_LIBS opencv
+            BOB_MODULES common imgproc robots video)
```

Comment:
I've ported the code to the new version of BoB robotics. *If* you can make this work for you then that'd probably be best because we've fixed a lot of bugs since the version you're using...